sh_script/build_final.sh

```diff
@@ -3,26 +3,34 @@
 export CC=clang
 export AR=llvm-ar
 
-if [[ ! $PWD =~ rust-td$ ]];then
+if [[ ! $PWD =~ td-shim$ ]];then
     pushd ..
 fi
 
-cargo xbuild -p rust-tdshim --target x86_64-unknown-uefi --release
-pushd rust-td-payload
-cargo xbuild --target x86_64-unknown-uefi --release
-popd
-cargo run -p rust-td-tool -- \
-    target/x86_64-unknown-uefi/release/ResetVector.bin \
-    target/x86_64-unknown-uefi/release/rust-tdshim.efi \
-    target/x86_64-unknown-uefi/release/rust-td-payload.efi \
-    target/x86_64-unknown-uefi/release/final.bin
+cargo xbuild -p td-shim --target x86_64-unknown-uefi --release --features=main,tdx --no-default-features
+
+final_pe() {
+    echo final-pe
+    cargo xbuild -p td-payload --target x86_64-unknown-uefi --release --features=main,tdx --no-default-features
```

Comment:
Why is `--no-default-features` need here?

Reply:
Compiling without --no-default-features will not report an error, but qemu will report bad address when running，Now qemu does not support the boot-kernel feature but td-shim enables boot-kernel feture by default, so this parameter is added.